boards/frdm-k64f/include/periph_conf.h

```diff
@@ -102,47 +103,17 @@ extern "C"
  * @name ADC configuration
  * @{
  */
-#define ADC_NUMOF                    (1U)
-#define ADC_0_EN                     1
-#define ADC_MAX_CHANNELS             6
+static const adc_conf_t adc_config[] = {
```

Comment:
yes, as this did not make any trouble, I would like to leave it like this - and have a more generic discussion about `static const xx` vars in the periph_conf.h, as this effects also non-adc device drivers.